### PR TITLE
Update Svelte to 1.23.4

### PR DESCRIPTION
**Pull request by zimme**
Self explanatory.

### CODE REVIEW

**Comment by zimme** [comment]
In the future I'm thinking we should maybe keep this package in sync with svelte, when it comes to versioning.

Svelte 1.35.0 would give this package 1.35.0_1, then when we do internal changes we just bump the last _1 part.

But we can't do this until svelte's version moves beyond the current version of our packages.

**Comment by klaussner** [comment]
I agree, but isn't that possible already?

**Comment by zimme** [comment]
> I agree, but isn't that possible already?

The last published version of `svelte:compiler` is `1.3.1` which is higher than the current svelte version which will cause problems.

**Comment by zimme** [comment]
What happens when they do a `meteor update` and there is a later version published?

**Comment by zimme** [comment]
Maybe we should put `svelte:compiler` at `1.3.2` instead of `1.4.0` so that we don't have to wait for svelte to reach `1.4.1` before we can publish under the new version schema.

**Comment by zimme** [comment]
Let's just keep `svelte:core`'s version in sync with svelte and this way we can start with that now as the latest version of svelte is at 1.23.4 and the latest `svelte:core` is at `1.2.1` so this shouldn't cause a problem with future updates.

**Comment by klaussner** [comment]
That's what I had in mind. 👌  But I think we should we also bump the version of `svelte:compiler` to 1.23.4_1.

**Comment by zimme** [comment]
We can't put `svelte:compiler` on a lower version than it is on now, because that will cause trouble with us having to recommend versions below the currently published versions of `svelte:compiler` and what happens once svelte reaches version 1.3.1 which is the current version of `svelte:compiler`? We can't publish another release with that version on atmosphere again.

**Comment by zimme** [comment]
we can't align the version of `svelte:compiler` with svelte until svelte is above the latest published version of `svelte:compiler`.

File: compiler-no-html/package.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ Package.describe({
 
 Package.registerBuildPlugin({
   name: 'svelte-compiler',
-  use: ['svelte:core@1.2.1'],
+  use: ['svelte:core@1.23.4_1'],
```
Comment (klaussner) [inline compiler-no-html/package.js]: This update shouldn't be necessary because the package is already deprecated at its current version (1.3.0).

Comment (zimme) [inline compiler-no-html/package.js]: The latest version of `svelte:compiler-no-htlm` is `1.2.1`, so a version bump here is necessary if we want to publish a new version that will be deprecated.

Comment (zimme) [inline compiler-no-html/package.js]: lol, I guess this PR is off a bit.

Comment (zimme) [inline compiler-no-html/package.js]: Just ignore everything I just wrote... Something broke when I was thinking 😝 

Anyways, we should still be using the latest version of the compiler even in this deprecated version, no?

**Comment by klaussner** [comment]
Maybe I'm misunderstanding you, but the latest published version of `svelte:compiler` is 1.3.1 and the current version of Svelte is 1.23.4, which is _greater_, so a bump of `svelte:compiler` to 1.23.4_1 shouldn't be a problem.

**Comment by zimme** [comment]
> Maybe I'm misunderstanding you, but the latest published version of svelte:compiler is 1.3.1 and the current version of Svelte is 1.23.4, which is greater, so a bump of svelte:compiler to 1.23.4_1 shouldn't be a problem.

You are right again, I think something's broken in my brain today...

File: compiler-no-html/package.js (hunk 1 above)
Comment (klaussner) [inline compiler-no-html/package.js]: Right, the update makes sense. I forgot that 1.3.0 isn't published yet. 😅

**Comment by zimme** [comment]
I'm good with these changes, merge at will.

**Comment by klaussner** [comment]
Squashed into https://github.com/meteor-svelte/meteor-svelte/commit/9c676393bbcbb5ec9ad32be7b3d4b71557cf562e.

**Pull request closed without merging**